tests/test_definitions/coexistance.py

```diff
@@ -0,0 +1,255 @@
+from tests.test_utilities.test_utils import APITestCase, SystemInfo
+
+test_count = 1
+
+coexistance_tests = {
+    "create_e131_device_1": APITestCase(
+        execution_order=(test_count := test_count + 1),
+        method="POST",
+        api_endpoint="/api/devices",
+        expected_return_code=200,
+        payload_to_send={
+            "type": "e131",
+            "config": {
+                "icon_name": "mdi:led-strip",
+                "center_offset": 0,
+                "refresh_rate": 62,
```

Comment:
Dont hardcode refresh_rate, since 62 is not always valid.

Reply:
good point, I have to get my head around min config confirmations

Reply:
actually I think this one is intended, can't we request anything, and the system will select, but we can't check the output? 

But also noting its valid that I don't need all the params as I can also assume that default will be selected for anything I don't state....

Reply:
I will trim to only code up the important ones...